### PR TITLE
chore: handle promise rejection

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
chore

**Did you add tests for your changes?**
No
**If relevant, did you update the documentation?**
No
**Summary**

Handle Promise rejection.

**Does this PR introduce a breaking change?**

**Other information**
No

### CODE REVIEW

**Review by gairTanm — DISMISSED** [review]

File: packages/webpack-cli/bin/cli.js [hunk 1]
```diff
@@ -20,6 +20,8 @@ if (packageExists('webpack')) {
 } else {
     promptInstallation('webpack', () => {
         error(`It looks like ${yellow('webpack')} is not installed.`);
-    });
+    })
+        .then(() => info(`${yellow('webpack')} was installed sucessfully.`))
+        .catch(() => error(`Action Interrupted, Please try once again or install ${yellow('webpack')} manually.`));
```
Comment (evilebottnawi) [inline packages/webpack-cli/bin/cli.js:25]: What is the error code? if we in `catch`?

Comment (snitin315) [reply packages/webpack-cli/bin/cli.js]: I think it should be `2`, updated.

**Review by evilebottnawi — APPROVED** [review]

**Review by anshumanv — APPROVED** [review]

File: packages/webpack-cli/bin/cli.js [hunk 2]
```diff
@@ -20,6 +20,11 @@ if (packageExists('webpack')) {
 } else {
     promptInstallation('webpack', () => {
         error(`It looks like ${yellow('webpack')} is not installed.`);
-    });
+    })
+        .then(() => info(`${yellow('webpack')} was installed sucessfully.`))
```
Comment (anshumanv) [inline packages/webpack-cli/bin/cli.js:24]: Can we have tests?

**Review by anshumanv — CHANGES_REQUESTED** [review]
One comment

File: packages/webpack-cli/bin/cli.js (hunk 2 above)
Comment (snitin315) [reply packages/webpack-cli/bin/cli.js:24]: CI already installs webpack before running tests.

**Review by anshumanv — APPROVED** [review]

**Pull request merged**